src/agent.rs

```diff
@@ -9,9 +9,11 @@ use mio_extras::channel;
 use mio_extras::channel::Receiver;
 use std::process::{Command, ExitStatus, Output, Stdio};
 use std::thread;
+use std::time::Duration;
 
 const SERVER: Token = mio::Token(1);
 const STATUS: Token = mio::Token(2);
+const ERR_CIPHER_BLACKLISTED: i32 = 89;
```

Comment:
Doesn't 89 mean unsupported?

Reply:
That's correct, the shims return 89 when an option is not supported.
I just "reused" this error code to make use of the logic that marks tests as "skipped" when 89 is returned, and labeled it "ERR_CIPHER_BLACKLISTED" because that is currently the only case in which interop produces this error code internally. Should I rename it or introduce a different numerical error code to avoid confusion?